### ISSUE TITLE
Request to update to test against most recent versions

### ISSUE THREAD

**Issue by e-avalanche** [ydaniv/django-rest-assured#44]
This is a very useful project!  But it notes: 
```
Tests run against:

    Django 1.6 - 1.11.
    Django REST Framework 2.4.3 - 3.6.
    Python 2.7, 3.3 - 3.6 (3.2 should work but is not tested).
```
Can you test against the most recent versions (Django 2.2, DRF 3.9, Python 3.7) as well and update?  Thank you!

**Comment by ydaniv** [comment]
Hey @e-avalanche, unfortunately I haven't been around python for something like 2 years now, so it's quite a problem for me to continue supporting this project.

Do you think you could give it a shot and send a PR?

**Comment by ydaniv** [comment]
Resolved via #45
